### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by dearrudam**

### CODE REVIEW

File: quarkus-jnosql-mongodb/pom.xml [hunk 1]
```diff
@@ -59,6 +60,17 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.javafaker</groupId>
```
Comment (otaviojava) [inline quarkus-jnosql-mongodb/pom.xml:69]: why a old version of Java faker?

https://mvnrepository.com/artifact/com.github.javafaker/javafaker/1.0.2

File: quarkus-jnosql-mongodb/pom.xml [hunk 2]
```diff
@@ -59,6 +60,17 @@
...
+      <artifactId>javafaker</artifactId>
+      <version>0.15</version>
```
Comment (otaviojava) [inline quarkus-jnosql-mongodb/pom.xml]:
```suggestion
      <version>1.0.2</version>
````

File: quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java [hunk 3]
```diff
@@ -22,7 +22,7 @@ public List<Developer> listAll() {
         return template.select(Developer.class).result();
     }
 
-    @Path("/byName")
+    @Path("/findByName")
```
Comment (otaviojava) [inline quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java]: In the rest API nomenclature, does it make sense?

```suggestion
    @Path("/")
```

File: quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java [hunk 4]
```diff
@@ -22,7 +22,7 @@ public List<Developer> listAll() {
...
     @GET
     public List<Developer> findByName(@QueryParam("name") String name) {
```
Comment (otaviojava) [inline quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java]: What happens if "name" is empty? Should we list all instead?

File: quarkus-jnosql-mongodb/pom.xml (hunk 1 above)
Comment (dearrudam) [reply quarkus-jnosql-mongodb/pom.xml:69]: Oh no... I'll fix it! Thanks, @otaviojava!

File: quarkus-jnosql-mongodb/pom.xml (hunk 2 above)
Comment (dearrudam) [reply quarkus-jnosql-mongodb/pom.xml]: I'm fixing it.

File: quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java (hunk 4 above)
Comment (dearrudam) [reply quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java]: Yeah... it was intended to be a sample code, but you're right... I'm fixing it. Thanks, @otaviojava

File: quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java (hunk 3 above)
Comment (dearrudam) [reply quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java]: I'm fixing it. Thanks, @otaviojava!

**Review by otaviojava — APPROVED** [review]

**Comment by dearrudam** [comment]
@otaviojava, I've changed the `quarkus-jnosql-mongodb` module to use the 1.0.1-SNAPSHOT version for now, but I'm committed to changing it when the 1.0.1 version is released.

**Comment by otaviojava** [comment]
I've scheduled:

https://projects.eclipse.org/projects/technology.jnosql/reviews/1.0.1-release-review

**Pull request merged**